dashboard/modules/job/job_manager.py

```diff
@@ -517,15 +520,18 @@ async def submit_job(
         # returns immediately and we can catch errors with the actor starting
         # up.
         try:
+            resources = None
+            if _driver_on_current_node:
```

Comment:
Should we also do it for the agent? So the actors will always start on the agent where the data (runtime env) exists.

Reply:
I think it is a reasonable requirement to set resources for the driver process in the future, so which node the driver should be launched should be decided by GCS/raylet.